Documentation/using-corert/optimizing-corert.md

```diff
@@ -37,3 +37,6 @@ No action is needed on Windows since the platform convention is to generate debu
 
 ## Advanced options 
 * `<IlcDisableUnhandledExceptionExperience>true</IlcDisableUnhandledExceptionExperience>`: disables code that prints stack traces for unhandled exceptions to the console.
+* `<IlcSystemModule>classlibmodule</IlcSystemModule>`: Name of the module which contains basic classes. When specified, disable automatic referencing of the `System.Private.CoreLib` and other libraries. 
+* `<IlcMultiModule>true</IlcMultiModule>`: Compile only input files (do not compile referenced assemblies)
+* `<ExperimentalDynamicCodeSupport>true</ExperimentalDynamicCodeSupport>`: **Experimental**: Ignore metadata blocking for internal implementation details. Maybe removed anytime.
```

Comment:
This is not a public option. This is set automatically when the interpreter/JIT are enabled. Let's not document.